routes/jobs.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+  getAllJobs,
+  getJob,
+  createJob,
+  updateJob,
+  deleteJob,
+} = require("../controllers/jobs");
+
+router.route("/").post(createJob).get(getAllJobs);
+router.route("/:id").get(getJob).delete(deleteJob).patch(updateJob);
```

Comment:
`deleteJob` and `updateJob` are not implemented. Also, as a nit, I'd recommend breaking each method up onto a separate line, like:

```
router.route('/:id')
  .get(...)
  .post(...)
  .delete(...)
```